substrate/frame/identity/src/lib.rs

```diff
@@ -925,7 +1179,99 @@ impl<T: Config> Pallet<T> {
 		fields: <T::IdentityInformation as IdentityInformationProvider>::FieldsIdentifier,
 	) -> bool {
 		IdentityOf::<T>::get(who)
-			.map_or(false, |registration| (registration.info.has_identity(fields)))
+			.map_or(false, |(registration, _username)| (registration.info.has_identity(fields)))
+	}
+
+	/// Calculate the deposit required for an identity.
+	fn calculate_identity_deposit(info: &T::IdentityInformation) -> BalanceOf<T> {
+		let bytes = info.encoded_size() as u32;
+		let byte_deposit = T::ByteDeposit::get().saturating_mul(<BalanceOf<T>>::from(bytes));
+		T::BasicDeposit::get().saturating_add(byte_deposit)
+	}
+
+	/// Validate that a username conforms to allowed characters/format. The function will validate
```

Comment:
```suggestion
	/// Validate that a username conforms to allowed characters/format.
	///
	/// The function will validate
```
Normally we do one sentence and then a newline followed by more explanations. AFAIK it renders nicer in the docs.